### ISSUE TITLE
Fault code SOAP-ENV:Client: Method 'n1:...' not implemented

### ISSUE THREAD

**Issue by Massimo-B** [KDAB/KDSoap#101]
There is some namespace misalignment when creating a client from
http://www.genivia.com/calc.wsdl

The client receives the error:
`Fault code SOAP-ENV:Client: Method 'n1:add' not implemented`

Parsing the XML request from KDSoap this XML is generated:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<soap:Envelope
    xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"
    xmlns:soap-enc="http://schemas.xmlsoap.org/soap/encoding/"
    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    >
  <soap:Body>
    <n1:add xmlns:n1="http://websrv.cs.fsu.edu/~engelen/calc.wsdl">
      <a xsi:type="xsd:double">5</a>
      <b xsi:type="xsd:double">5</b>
    </n1:add>
  </soap:Body>
</soap:Envelope>

```

After comparing with the gSOAP generated XML request, sending a slightly modified XML request via curl makes it work:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<soap:Envelope
    xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/"
    xmlns:soap-enc="http://schemas.xmlsoap.org/soap/encoding/"
    xmlns:xsd="http://www.w3.org/2001/XMLSchema"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:n1="urn:calc"
    >
  <soap:Body>
    <n1:add soap:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/">
      <a xsi:type="xsd:double">5</a>
      <b xsi:type="xsd:double">5</b>
    </n1:add>
  </soap:Body>
</soap:Envelope>
```

**Comment by dfaure-kdab** [comment]
I had a look and the issue seems to be that kdwsdl2cpp ignores the namespace attribute of SOAP:body.

    <SOAP:body use="encoded" namespace="urn:calc">

Given that the operation "add" is defined in the targetNamespace "http://websrv.cs.fsu.edu/~engelen/calc.wsdl", that's what KDSoap uses when generating the "add" element.

Strangely enough, this feature hasn't been used by any of the WSDL files given to KDSoap until now... well, it only applies to RPC style calls I suppose.

Currently working on a fix, please stay tuned.

**Comment by dfaure-kdab** [comment]
Fixed in commit 3652412.
